app/i18n/cz/gen.php

```diff
@@ -144,6 +144,7 @@
 		'en' => 'English',	// IGNORE
 		'en-us' => 'English (United States)',	// IGNORE
 		'es' => 'Español',	// IGNORE
+		'fa' => 'فارسی',
```

Comment:
Presumably you'd want to ignore this particular string everywhere. (That can be done through the CLI or just some global replace.) Without `	// IGNORE` at the end the tools will say it's a string that needs translating.